src/App.test.jsx

```diff
@@ -1,4 +1,15 @@
+import { findByText, getByAltText, getByText, render, screen } from '@testing-library/react'
```

Comment:
These `finBy `and `getBy `functions do not need to be imported separately as they can be accessed from `screen`